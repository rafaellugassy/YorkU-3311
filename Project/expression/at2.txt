quotient 

restart

quotient
int_value (5)
int_value (0)
analyze
simplify

restart