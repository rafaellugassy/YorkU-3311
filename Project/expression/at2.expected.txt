  Expression currently specified: ?
  Report: Expression is initialized.
->quotient
  Expression currently specified: (? / nil)
  Report: OK.
->restart
  Expression currently specified: ?
  Report: OK.
->quotient
  Expression currently specified: (? / nil)
  Report: OK.
->int_value(5)
  Expression currently specified: (5 / ?)
  Report: OK.
->int_value(0)
  Expression currently specified: (5 / 0)
  Report: OK.
->analyze
  Expression currently specified: (5 / 0)
  Report: (5 / 0) is type-correct.
->simplify
  Expression currently specified: (5 / 0)
  Report: Error (Divisor is zero).
->restart
  Expression currently specified: ?
  Report: OK.
